src/error.rs

```diff
@@ -44,6 +43,8 @@ pub enum Error {
     ExpectedPositiveValue,
     #[cfg(feature = "kurbo")]
     ConvertContour(ErrorKind),
+    MissingFile(String),
+    MissingUFODir(String),
```

Comment:
Rust naming convention is to only capitalize the first letter of acronyms and initialisms 🤷 
```suggestion
    MissingUfoDir(String),
```

Reply:
Ty!

7b6cb1e672f5ec81ef155044f3d5f50080d6dec0